### ISSUE TITLE
Change community site from "Du" to "Sie"?

### ISSUE THREAD

**Issue by Ein-Tim** [corona-warn-app/cwa-website#2400]
## Your Question
Since version 2.17, the user finds a reference to the community page in the app under "More" / 3 dots -> "Social Media".
The app always uses "Sie", however, the community page uses "Du".
So currently, the user is transferred from the app which always uses "Sie" to a web page which uses "Du". 
Should this be changed?

**Comment by MikeMcC399** [comment]
@Ein-Tim 

I don't think that this should be changed. In the cases on GitHub, where German is used in community conversation, going by memory I think that most conversations use "Du".

Otherwise most of the community interaction in GitHub is in English where the problem does not arise. In English it is always "you". 

In German the page https://www.coronawarn.app/de/community/ uses "Du". This page links to:
- Social media https://www.coronawarn.app/de/community/#social-media
- https://twitter.com/coronawarnapp is using "Ihr / Euch"
- https://www.instagram.com/rki_fuer_euch/ seems to avoid addressing the reader personally and the channel is a mixture of different topics, not specifically COVID-19 or CWA
- https://www.coronawarn.app/de/faq/#support which use "Sie"
- https://github.com/corona-warn-app where the README files of the individual repositories are in English, except for https://github.com/corona-warn-app/cwa-documentation/blob/main/translations/README.de.md which is an alternative German-language version.

The page also links to several English language pages where "Du" / "Sie" is not relevant:
- https://github.com/corona-warn-app/cwa-documentation/blob/main/CODE_OF_CONDUCT.md
- https://github.com/corona-warn-app/cwa-documentation/blob/main/LICENSE
- https://github.com/corona-warn-app/cwa-documentation/blob/main/CODEOWNERS

It would not be wrong to change the Community page to use "Sie", but on the other hand it might create a barrier for technical users if they thought that interactions with the developers would be more formal. So it is a decision which requires a value judgement.

**Comment by dsarkar** [comment]
@MikeMcC399 @Ein-Tim 
I don't know what the motives were when the Community page was created and it was decided to use"Du"-form. However, I think there several reasons that explain the use of "Du" particularly on the Community page:

The "Du" form reflects the changes in professional life in the age of digitalization:

- Underlines the high value of a strong sense of we
- Expression of a new trustful togetherness
- Encouraging each individual to contribute his or her own ideas, at eye level

**Comment by MikeMcC399** [comment]
@dsarkar 
I fully agree with you. I think using "Du" on the Community page is good!

**Comment by Ein-Tim** [comment]
I agree with both of you and als think that "Du" is better in this case. I just wanted to ask to make sure that this is no problem  for the product owner or the stakeholders.
